spring-boot-dependencies/pom.xml

```diff
@@ -135,6 +135,7 @@
 		<spring-amqp.version>1.6.0.M1</spring-amqp.version>
 		<spring-cloud-connectors.version>1.2.1.RELEASE</spring-cloud-connectors.version>
 		<spring-batch.version>3.0.6.RELEASE</spring-batch.version>
+		<spring-data-geode.version>1.7.0.APACHE-GEODE-EA-M1</spring-data-geode.version>
```

Comment:
Unrelated but that's a weird version naming. Any reason why it has to be that long?